src/Controller/Controller.php

```diff
@@ -572,6 +572,9 @@ public function render($view = null, $layout = null)
             return $this->response;
         }
 
+        // Re-fetch View class to pass view variables set in beforeRender callbacks
+        $this->View = $this->getView();
+
         if ($viewClass !== $this->viewClass) {
             $this->View = $this->getView($this->viewClass);
```

Comment:
I'm concerned about this conditional here. Does this mean that if another view is instantiated all configuration that the other view had is lost?